tests/globalkilltest/global_kill_test.go

```diff
@@ -302,9 +315,13 @@ func (s *TestGlobalKillSuite) killByKillStatement(c *C, db1 *sql.DB, db2 *sql.DB
 	var connID2 uint64
 	err = conn2.QueryRowContext(ctx, "SELECT CONNECTION_ID();").Scan(&connID2)
 	c.Assert(err, IsNil)
-	log.Infof("connID2: 0x%x", connID2)
+	log.Info("connID2", zap.String("connID2", "0x"+strconv.FormatUint(connID2, 16)))
 
-	log.Infof("exec: KILL QUERY %v(0x%x) [on 0x%x]", connID1, connID1, connID2)
+	log.Info("exec: KILL QUERY",
+		zap.Uint64("connID1 value", connID1),
+		zap.String("connID1", "0x"+strconv.FormatUint(connID1, 16)),
+		zap.String("on connID2", "0x"+strconv.FormatUint(connID2, 16)),
+	)
```

Comment:
```go
log.Info("exec: KILL QUERY",
    zap.String("connID1", "0x"+strconv.FormatUint(connID1, 16)),
    zap.String("connID2", "0x"+strconv.FormatUint(connID2, 16)),
)
```

Reply:
Suggestions above is accepted and code is changed in the last commition. But 
idc-jenkins-ci-tidb/check_dev_2 is failed which seems not the code problem. Could you reload the testing task?
```
[2021-03-11T00:46:20.427Z] --- FAIL: TestInfo (0.33s)
[2021-03-11T00:46:20.427Z]     domain_test.go:210: etcdserver: requested lease not found
```

Reply:
All tests checked, and code is changed :-)